### PR TITLE
[JENKINS-34848] Download gradle instead of including it in the repo

### PR DESCRIPTION
**Pull request by andresrc**
Amends https://github.com/jenkinsci/jenkins-test-harness-tools/pull/3 to avoid maintaining the file in the git repo.

@reviewbybees esp. @jglick

### CODE REVIEW

**Comment by aheritier** [comment]
You can in that case do the same for maven from my POV @andresrc

**Comment by andresrc** [comment]
yes, if the approach seems ok I can add it to the remaining files, of course.

**Comment by ghost** [comment]
This pull request originates from a [CloudBees](https://www.cloudbees.com/) employee. At CloudBees, we require that all pull requests be reviewed by other CloudBees employees before we seek to have the change accepted. If you want to learn more about our process please see [this explanation](https://github.com/reviewbybees/about#about-reviewbybees).

File: pom.xml [hunk 1]
```diff
@@ -138,6 +138,24 @@ THE SOFTWARE.
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>com.googlecode.maven-download-plugin</groupId>
+                <artifactId>download-maven-plugin</artifactId>
```
Comment (jglick) [inline pom.xml:143]: https://discuss.gradle.org/t/host-gradle-wrapper-distributions-on-maven-central/543/2 *sigh*

**Review by jglick — APPROVED** [review]
As noted in the readme, I am not sure this is going to be enough; we may need to rewrite this repo. But worth a shot.

**Comment by jglick** [comment]
Does not suffice. The artifact as currently conceived is literally unreleasable. Needs to be deprecated.

**Pull request merged**